pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -165,20 +165,6 @@ const fluentConfTemplate = `{{- define "fluentConf" }}
 		ssl_partial_chain "#{ENV['SSL_PARTIAL_CHAIN'] || 'true'}"
 	</filter>
 		
-	<filter kubernetes.journal.**>
-		@type parse_json_field
-		merge_json_log "#{ENV['MERGE_JSON_LOG'] || 'false'}"
-		preserve_json_log "#{ENV['PRESERVE_JSON_LOG'] || 'true'}"
-		json_fields "#{ENV['JSON_FIELDS'] || 'MESSAGE,log'}"
-	</filter>
-		
-	<filter kubernetes.var.log.containers.**>
-		@type parse_json_field
-		merge_json_log "#{ENV['MERGE_JSON_LOG'] || 'false'}"
-		preserve_json_log "#{ENV['PRESERVE_JSON_LOG'] || 'true'}"
-		json_fields "#{ENV['JSON_FIELDS'] || 'log,MESSAGE'}"
-	</filter>
-
```

Comment:
can you remind me again what this was doing and why its now alright to remove it? is this only for the `merge_json` case?

Reply:
I'll add these back in since 'preserve_json_log' is true and we set the env to true.  I dont recall what this plugin does

Reply:
From the [Fluend plugin doc](https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter):
NOTE: As of the release 2.1.x of this plugin, it no longer supports parsing the source message into JSON and attaching it to the payload. The following configuration options are removed:
- merge_json_log
- preserve_json_log

Reply:
resolving as I reduced the scope of the PR